lib/Runtime/Math/WasmMath.inl

```diff
@@ -127,10 +127,16 @@ inline int WasmMath::Eqz(T value)
     return value == 0;
 }
 
-template<typename T>
-inline T WasmMath::Copysign(T aLeft, T aRight)
+template<>
+inline double WasmMath::Copysign(double aLeft, double aRight)
+{
+    return copysign(aLeft, aRight);
```

Comment:
The Linux builds are complaining about _copysignf. Do you know of an alternative ?